XSUB.h

```diff
@@ -159,35 +159,36 @@ is a lexical C<$_> in scope.
 
 #define dAX const I32 ax = (I32)(MARK - PL_stack_base + 1)
 
-#define dAXMARK                         \
```

Comment:
Ah, i guess you are being thrown off because this step includes two steps, "break multiline define" and "fixup line continuations and related ws", as do most of the steps following it. I should break the first one out to its own step. "Break multiline define" means take multiline defines which have a value (replacement) segment on the define line itself, and move that first line down. So this

```
#define Foo(x,y,z) STMT_START { \
```
turns into this:
```
#define Foo(x,y,z) \
    STMT_START { \
```
as this affects line continuations it is followed by a second step which cleans up the line continuations. But the first time the line continuations are changed should be an explicit step.